local/rest_api_gcbm/preprocess.py

```diff
@@ -153,15 +149,15 @@ def get_input_layers():
             ) as json_file:
                 dictionary["layer_type"] = "GridLayer"
                 dictionary["layer_data"] = "Int16"
-                dictionary["nodata"] = nodatam[1]
+                dictionary["nodata"] = 32767
```

Comment:
@Namyalg @Janvi-Thakkar @SanjaySinghRajpoot @YashKandalkar  - what do you think of this solution?

Reply:
I think, as a temporary fix, we can probably go ahead with this

Reply:
We might also need a comment explaining this temporary solution, so that other contributors know why the hard coded value is there.

Reply:
@Freeman-kuch - can you please add documentation to this patch? Using a hard coded number is not a good long term solution, but it will do for now.

Reply:
I mean leaving a comment in the code to explain your change. Without documentation it's hard for others to understand why this value is set.

Maybe something like:

> # FIXME: hard coded nodata value is a temporary solution, but should instead come from the input data source.